src/lib.rs

```diff
@@ -13,58 +12,58 @@ extern crate rustc_data_structures;
 extern crate rustc_mir;
 extern crate rustc_target;
 
+mod fn_call;
+mod operator;
+mod intrinsic;
+mod helpers;
+mod tls;
+mod range_map;
+mod mono_hash_map;
+mod stacked_borrows;
+
 use std::collections::HashMap;
 use std::borrow::Cow;
 
 use rustc::ty::{self, TyCtxt, query::TyCtxtAt};
 use rustc::ty::layout::{LayoutOf, Size, Align};
 use rustc::hir::{self, def_id::DefId};
 use rustc::mir;
-
+pub use rustc_mir::interpret::*;
+// Resolve ambiguity.
+pub use rustc_mir::interpret::{self, AllocMap, PlaceTy};
 use syntax::attr;
 use syntax::source_map::DUMMY_SP;
 
-pub use rustc_mir::interpret::*;
-pub use rustc_mir::interpret::{self, AllocMap, PlaceTy}; // resolve ambiguity
-
-mod fn_call;
-mod operator;
-mod intrinsic;
-mod helpers;
-mod tls;
-mod range_map;
-mod mono_hash_map;
-mod stacked_borrows;
-
 pub use crate::fn_call::EvalContextExt as MissingFnsEvalContextExt;
 pub use crate::operator::EvalContextExt as OperatorEvalContextExt;
 pub use crate::intrinsic::EvalContextExt as IntrinsicEvalContextExt;
 pub use crate::tls::{EvalContextExt as TlsEvalContextExt, TlsData};
 use crate::range_map::RangeMap;
-#[allow(unused_imports)] // FIXME rustc bug https://github.com/rust-lang/rust/issues/53682
+// FIXME: rustc bug, issue <https://github.com/rust-lang/rust/issues/53682>.
+#[allow(unused_imports)]
```

Comment:
I find it clearer with both on the same line, that was deliberate.

When they are on separate lines, it gets harder to see what the scope of the comment is, i.e. where the code it refers to ends.

Reply:
I'm definitely of a different opinion, but if this makes you happy, sure.